cnsi-inspector/pkg/image-scanner/controller.go

```diff
@@ -143,6 +230,12 @@ func (c *controller) Run(ctx context.Context, policy *v1alpha1.InspectionPolicy)
 		NamespaceAssessments: make([]*itypes.NamespaceAssessment, 0),
 	}
 
+	// GetVacAdapter
+	vacAdapter, err := c.GetVacAdapter(setting)
```

Comment:
I think this should be called only if VacAssessment is enabled.

Reply:
it is inline with other adapter implementation(harbor)
Adapter will generate the Assessment. 

This is not the VacAssessment, it is just adapter to fetch the VacAssessment

Reply:
True, it should have AssessmentEnabled check